app/src/main/java/org/simple/clinic/facility/change/FacilityListItem.kt

```diff
@@ -6,8 +6,9 @@ import org.simple.clinic.facility.Facility
 sealed class FacilityListItem {
```

Comment:
Reword commit to `Improve logic for toggling space between sections in FacilitiesAdapter`?

Reply:
Made this change.